guides/aws/cloud-practitioner/README.md

```diff
@@ -0,0 +1,107 @@
+  * What is meant by AWS Cloud security and compliance concepts?
+  * What are the AWS Identity & Access Management (IAM) capabilities?
+  * What are the resources for security support?
+
+* **Domain 3:** Technology (36%)
+  * What methods can be used to deploy and operate in the AWS Cloud?
+  * What is meant by the AWS global infrastructure?
+  * What are the core AWS services?
+  * What are the resources for technology support?
+
+* **Domain 4:** Billing and Pricing (12%)
+  * What are the differences between the different AWS pricing models?
+  * What are the various AWS account structures?
+  * What are the resources for billing and pricing support?
+
+#### Resources
+
+To answer the questions above, we have compiled a list of resources that we think are useful for the AWS Certified Cloud Practitioner exam. 
+
+It is important to note that there are different methodologies of studying. Hence, choose whichever resources you think would work best for you.
+
+##### AWS Whitepapers
+* [Overview of Amazon Web Services](https://d0.awsstatic.com/whitepapers/aws-overview.pdf)
+* [Architecting for the Cloud: AWS Best Practices](https://d1.awsstatic.com/whitepapers/AWS_Cloud_Best_Practices.pdf)
+* [How AWS Pricing Works](http://d1.awsstatic.com/whitepapers/aws_pricing_overview.pdf)
+* [Compare AWS Support Plans](https://aws.amazon.com/premiumsupport/plans)
+
+##### Courses/Videos
+* [AWS Cloud Practitioner Essentials (Second Edition)](https://www.aws.training/Details/Curriculum?id=27076)
+* [A Cloud Guru - AWS Certified Cloud Practitioner 2019](https://acloud.guru/learn/aws-certified-cloud-practitioner)
+* [LinuxAcademy - AWS Cloud Practitioner (CLF-C01)](https://linuxacademy.com/course/aws-cloud-practitioner)
+
+##### Practice Tests
+
+We found the following list of practice tests to be useful and recommend attempting all of them to gain a solid understanding of the AWS Certified Cloud Practitioner.
+
+* [A Cloud Guru - AWS Certified Cloud Practitioner Exam](https://acloud.guru/exam-simulator/start?courseId=aws-certified-cloud-practitioner)
+* [AWS Certified Cloud Practitioner – Foundational (CLF-C01) Sample Exam Questions](https://d1.awsstatic.com/training-and-certification/Docs%20-%20Cloud%20Practitioner/AWS%20Certified%20Cloud%20Practioner_Sample%20Questions_v1.1_FINAL.PDF)
+* [AWS Cloud Practitioner Essentials (Second Edition): Final Course Knowledge Check](https://www.aws.training/Details/eLearning?id=37545)
+* [LinuxAcademy - AWS Certified Cloud Practitioner](https://beta.linuxacademy.com/#/challenges/details/25ef6f9b-cf3e-4bd4-aac4-4cc334584698)
+* [Whizlabs - AWS Certified Cloud Practitioner](https://www.whizlabs.com/learn/course/aws-certified-cloud-practitioner-practice-tests)
+
+##### Websites
+
+* [AWS Newbies - Certified Cloud Practitioner Exam](https://awsnewbies.com/cloud-practitioner)
+
+### Schedule your exam
+
+#### Exam information
+* Name: AWS Certified Cloud Practitioner
+* Cost: 100 USD (100 GBP incl. VAT)
```

Comment:
I booked my exam with Pearson VUE yesterday, it was $100 + VAT, so $120 total.

Reply:
The VAT rate differs with country and the 100 USD is with reference to the cost of the exam only. I have only included the VAT for our GBP currency as I thought this makes most sense for our internal users.